### PR TITLE
Update to 1.2

### PR DESCRIPTION
**Pull request by AsgerStage**
- Added event-subscription endpoints
  - Endpoint for creating subscription
  - Endpoint for deleting subscription
  - Endpoint for updating subscription
  - Endpoint for receiving list of subscriptions
  - Endpoint for reading a single subscription
- Added eventType discriminator
- Added endpoint for reading a single event by eventID

### CODE REVIEW

File: v1/swagger.yaml [hunk 1]
```diff
@@ -161,6 +161,7 @@ paths:
           name: subscriptionID
           schema:
             type: string
+            format: uuid
           required: true
           description: The ID of the subscription to receive
```
Comment (HenrikHL) [inline v1/swagger.yaml]: Maybe change the wording to :
The universal unique ID of the subscription to receive
?

File: v1/swagger.yaml [hunk 2]
```diff
@@ -555,6 +549,16 @@ components:
       items:
         $ref: '#/components/schemas/eventType'
 # Added by API Auto Mocking Plugin
+  parameters:    
+    subscriptionIDPathParam:
+      name: subscriptionID
+      in: path
+      description: The unique ID associated with the subscription.
```
Comment (HenrikHL) [inline v1/swagger.yaml:557]: add **universal** unique...

File: v1/swagger.yaml [hunk 3]
```diff
@@ -524,7 +524,8 @@ components:
         subscriptionID:
           type: string
           format: uuid
-          example: jlke123j
+          example: 123e4567-e89b-12d3-a456-426614174000
+          nullable: false
```
Comment (HenrikHL) [inline v1/swagger.yaml:528]: Betyder dette at det er en required property? Hvis det er i forbindelse med et POST kald fra en subscriber så er det forkert - da det er publisher som laver subscriptionID'erne...?

File: v1/swagger.yaml (hunk 1 above)
Comment (AsgerStage) [reply v1/swagger.yaml]: done

File: v1/swagger.yaml (hunk 3 above)
Comment (AsgerStage) [reply v1/swagger.yaml]: har fjernet nullable

**Pull request closed without merging**